src/engine/ruleset/integrations/docker-listener/documentation.yml

```
@@ -0,0 +1,19 @@
+title: Wazuh Docker Listener Integration
+overview: >
+  This integration processes logs obtained with the docker-listener wodle from docker.
+  This are real-time events from the docker server.
+compatibility: >
+  This module has been tested against version 20.10.12
+configuration: >
+  Event are collected setting the ossec in the agent or the manager in the following way:
+
+  ```xml
+  <wodle name="docker-listener">
+      <interval>10m</interval>
+      <attempts>5</attempts>
+      <run_on_start>yes</run_on_start>
+      <disabled>no</disabled>
+  </wodle>
+  ```
+event:
+  module: docker-listener
```

Missing event.dataset
```suggestion
  module: docker-listener
  dataset: woodle-docker-listener
```